### PR TITLE
Add Open-LLaMA instructions

### LINKED ISSUE #224: Using open-llama instead of LLaMA
I have tried to convert open-llama from https://huggingface.co/openlm-research/open_llama_7b_preview_200bt/tree/main, converting from transformers version to "normal" using convert_hf_checkpoint and then convert_checkpoint, it failed in later one at:

```
  converted["transformer.wte.weight"] = state_dict["tok_embeddings.weight"].to(dtype)
  converted["lm_head.weight"] = state_dict["output.weight"].to(dtype)
  converted["transformer.ln_f.scale"] = state_dict["norm.weight"].to(dtype)
``` 

Keys tok_embeddings.weight, output.weight and norm.weight are missing in state_dict. I tried avoiding that stating

`converted = state_dict`

but I got nothing, inference was broken. I am pretty much sure I was shooting in a dark with an empty gun. Any help is appreciated.

### PR DESCRIPTION
**Pull request by lantiga**
Fixes #224

### CODE REVIEW

**Review by carmocca — APPROVED** [review]
Nice!

**Comment by lantiga** [comment]
Thanks for the fix @carmocca

File: howto/download_weights.md [hunk 1]
```diff
@@ -27,6 +27,59 @@ python scripts/convert_checkpoint.py --model_size 7B
 > **Note**
 > All scripts support argument [customization](customize_paths.md)
 
+### OpenLLaMA
+
+OpenLM Research has released **Apache 2.0 licensed** weights obtained by training LLaMA on the RedPajama dataset.
```
Comment (rasbt) [inline howto/download_weights.md:32]: For people not familiar with that, maybe change

> RedPajama dataset.

to 

> the 1.2 trillion token open-source [RedPajama dataset](https://github.com/togethercomputer/RedPajama-Data).

Comment (lantiga) [reply howto/download_weights.md:32]: Good point

File: scripts/convert_hf_checkpoint.py [hunk 2]
```diff
@@ -17,21 +17,19 @@
 @torch.no_grad()
 def convert_hf_checkpoint(
     *,
-    output_dir: Path = Path("checkpoints/lit-llama"),
-    ckpt_dir: Path = Path("checkpoints/hf-llama/"),
+    output_dir: Path = Path("checkpoints/lit-llama/7B"),
+    checkpoint_dir: Path = Path("checkpoints/hf-llama/7B"),
     model_size: str = "7B",
```
Comment (rasbt) [inline scripts/convert_hf_checkpoint.py:22]: Do the new weights support other models than 7B, if not, maybe adding "not supported yet" when other sizes are used

Comment (lantiga) [reply scripts/convert_hf_checkpoint.py:22]: We could add an internal check on weights sizes in a future PR

Comment (rasbt) [reply scripts/convert_hf_checkpoint.py:22]: ok happy to do that later

**Pull request merged**